Assets/Scripts/Controllers/CharacterController-DESKTOP-7IQNCQ4.cs

```diff
@@ -28,8 +31,15 @@ public void OnMove(InputValue value)
         _movement = value.Get<Vector2>();
     }
 
+    public bool onGround()
+    {   
+        RaycastHit2D hit = Physics2D.Raycast(_cc.bounds.center, Vector2.down, _cc.bounds.extents.y + 0.1f,groundMask);
```

Comment:
This is a pretty clever way of determining the raycast position, nice!

No action needed.